app/celery/letters_pdf_tasks.py

```diff
@@ -133,39 +133,44 @@ def collate_letter_pdfs_to_be_sent():
 
     letters_to_print = get_key_and_size_of_letters_to_be_sent_to_print(print_run_deadline)
 
-    for i, letters in enumerate(group_letters(letters_to_print)):
-        filenames = [letter['Key'] for letter in letters]
-
-        hash = urlsafe_b64encode(sha512(''.join(filenames).encode()).digest())[:20].decode()
-        # eg NOTIFY.2018-12-31.001.Wjrui5nAvObjPd-3GEL-.ZIP
-        dvla_filename = 'NOTIFY.{date}.{num:03}.{hash}.ZIP'.format(
-            date=print_run_deadline.strftime("%Y-%m-%d"),
-            num=i + 1,
-            hash=hash
-        )
+    i = 0
+    for zip_folder, letters_list in letters_to_print.items():
+        for letters in group_letters(letters_list):
+            i += 1
+            filenames = [letter['Key'] for letter in letters]
+
+            hash = urlsafe_b64encode(sha512(''.join(filenames).encode()).digest())[:20].decode()
+            # eg NOTIFY.2018-12-31.001.Wjrui5nAvObjPd-3GEL-.ZIP
+            dvla_filename = 'NOTIFY.{date}.{num:03}.{hash}.ZIP'.format(
```

Comment:
I wanted to include postage name in ZIP files, but Pete told me that I should not change zip file names at all, because scripts at DVLA rely on them having the name structure they have now.

Reply:
ok, I contacted DVLA and they are fine with this after all, so I applied those changes in my latest commit.

DVLA asked to first send them those zipfiles with new names to test environment. Would you know how to do that? ☺️

Reply:
their test environment == our staging environment, i believe. so can send stuff from there. either manually (try sshing on to the notify-ftp box on staging and using the creds in the env to connect, then sending some zip files with the proposed name). or try deploy to staging and send some letters and trigger.

Reply:
Ok, I think I will try to deploy to staging from a branch and send some letters from there tomorrow 👍

Reply:
ok DVLA are fine with new names and only ask  to be notified once we are ready to push those live.